src/components/helper/HelperWindow.vue

```diff
@@ -15,15 +15,16 @@
         <button class="button is-primary" @click="$refs.modalCard.hide()" v-translate>
           Close
         </button>
-        <a
-          v-if="$user.isLogged && showModifyButton"
-          class="button is-info"
-          target="_blank"
-          rel="noopener"
-          :href="'https://www.camptocamp.org/articles/' + helper.documentId"
-          v-translate>
-          Edit
-        </a>
+        <router-link
```

Comment:
Can't remember why I used an anchor instead of a router-link component, so I guess it's okay. Just twwo point : 

* Put the `v-if` on router link (otherwise, an invisible anchor is present)
* @brunobesson you add `noopener` attribute, is it relevant here ?

Reply:
@cbeauchesne no it's ok, because we link to our site. Still, there is nothing bad adding it, so I would prefer to add it anyway.

Reply:
If there is a good reason to add it, we'll make a separate PR to add it on all `<router-link>` . For just this PR, IMO, it does not make sense to add it on only one link.